code/modules/research/destructive_analyzer.dm

```diff
@@ -39,7 +39,7 @@
 		. += span_notice("An item can be loaded inside via [EXAMINE_HINT("Left-Click")].")
 
 /obj/machinery/rnd/destructive_analyzer/base_item_interaction(mob/living/user, obj/item/weapon, list/modifiers)
-	if(user.combat_mode)
+	if(weapon.tool_behaviour || user.combat_mode)
```

Comment:
Wouldn't this hard-block the majority of alien tools from being put in the destructive analyzer? And thus from being deconstructed and researched

Reply:
Do they have `tool_behaviour`? Maybe i'll hard check just for screwdriver then